### DISCUSSION TITLE
GoldHEN Cheats Manager v1.1.4

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/PS4CheatsManager#107] [Announcements]
### Added

- Add Plugins updater
- Backup Local Plugins
- Improve local cheats & patches updater
- Local update (Cheats, Patches, Plugins) scans files in 8 possible mounted USB partitions and HDD
- Adjust Circle/Cross button assignment based on console settings
- Download App update .pkg to `usb0` if available

### Changed

- Changed backup paths to:
  - `/mnt/usb0/backups/(name)`
  - `/data/GoldHEN/backup/name/`

### Misc

- Update Open Orbis SDK version
- Includes latest official cheat+patch+plugin pack (2023-09-21)

<hr /><em>This discussion was created from the release <a href='https://github.com/GoldHEN/GoldHEN_Cheat_Manager/releases/tag/v1.1.4'>GoldHEN Cheats Manager v1.1.4</a>.</em>

**Comment by RetrogamerX** [comment]
I can't get games to show up?

**Reply by bucanero** [reply]
not sure what you mean with "can't get games to show up", but just in case re-download the .pkg and install the 1.1.4 release again, since it was recently re-uploaded with a new build.

**Reply by RetrogamerX** [reply]
Yes it wasn’t showing my games in the goldhen app.  I will try newer version.  Ok cheers

**Comment by RetrogamerX** [comment]
> not sure what you mean with "can't get games to show up", but just in case re-download the .pkg and install the 1.1.4 release again, since it was recently re-uploaded with a new build.

Working great now. Thank you for all your hard work :)

**Reply by MusArk0321** [reply]
how do u fix? still can't see filter game i install.

i uninstall the GoldHEN Cheat Manager then reinstall it again but still not work.

**Comment by reyforce** [comment]
I downloaded the update from a previous version of the app and uninstalled it, then i used ps4xplorer to install the latest version of the goldhen cheats manager pkg. However i am constantly getting error when trying to install it. It says Install Failed with code 0x80990086. I tried installing via the homebrew store same error, tried with internal package installer again same error. I cant even find any info on this error.

**Reply by reyforce** [reply]
its working now just needed to restart the ps4 and the reinstall it via the homebrew store.

**Reply by RetrogamerX** [reply]
I uninstalled the old one and deleted the goldhen folder in the data folder.  Then install the newest version.   It seems to be fixed now.  Works great so far

**Comment by iehab-1985** [comment]
any cheats for lies of p

**Comment by fovdj** [comment]
Please help me, I just installed Goldhen Cheat manager 1.14 and update cheat database and patch, and check the cheat for the game, and it show up

But when I want to activate the cheat , it says the title for game not found..

Please help me...

Nb : FW 9.00 With Hen Karo 2.2.2
![20230929_063019](https://github.com/GoldHEN/GoldHEN_Cheat_Manager/assets/20830038/595d07b2-77bc-4a30-ae72-3270245c5335)
![20230929_063120](https://github.com/GoldHEN/GoldHEN_Cheat_Manager/assets/20830038/6bffc5f7-2588-46ca-aa55-0520b219b1e9)
![20230929_063253](https://github.com/GoldHEN/GoldHEN_Cheat_Manager/assets/20830038/e52fd7b5-eb86-41d5-abe6-d207da219766)

**Reply by fovdj** [reply]
Problem SOLVED, just update host with v.2.4b14

**Comment by lumine-babu-paimon** [comment]
Please update cheat Lie of P BANG

**Reply by RetrogamerX** [reply]
Give them time.  They will at some point

**Comment by ADAM37638** [comment]
Please help me, I just installed Goldhen Cheat manager 1.14 and update cheat database and patch, and check the cheat for the game, and it show up

But when I want to activate the cheat , it says the title for game not found.. Game one piece pirate warriors 4

Please help me...![image](https://github.com/GoldHEN/GoldHEN_Cheat_Manager/assets/149854911/195d87df-578a-48f1-99b3-cb55de3638b2)![image](https://github.com/GoldHEN/GoldHEN_Cheat_Manager/assets/149854911/171d0882-1a5f-403b-b716-b7adb5b80408)

**Comment by Ahmunet** [comment]
Hello I am unbale to updates the cheats and the patches, it doesn't work. I have restarted the ps4, reinstalled the cheats manager to no avail.
anyone can help?

**Comment by Shiroyasha-jzy** [comment]
Please make cheat for Red Dead Redemption 2 USA region V 1.29 pleasee:)

**Comment by royalbluewun** [comment]
is there a discord or site where we can request cheats? looking for Arkham knight cheats for 1.15

**Reply by bucanero** [reply]
Check this post https://github.com/GoldHEN/GoldHEN_Cheat_Repository/discussions/42

**Comment by banman2724** [comment]
Hey guys I'm having a big issue I upgraded to an SSD before that my plugins were working perfect cheap manager was working perfect patches an plugins everything. Well now since the update of me upgrading to an SSD. I cannot load any plugins. I cannot load any cheats. But I can load patches flawlessly. I don't know what to do can someone please help me I've been trying to fix it for the last 2 weeks. With no luck PLS someone help me
